docs/spec_draft.md

```diff
@@ -1461,6 +1462,89 @@ More formally, `result[i0, ..., iR-1] = operand[j0, ..., jR-1]` where
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.sort
+
+### Semantics
+
+Sorts a variadic number of tensors in `inputs` together, according to a custom 
+`comparator`, along the given `dimension` and produces a variadic number of
+tensors as `results`. If `is_stable` is true, then the sorting is stable, that
+is, relative order of elements considered to be equal by the comparator is
+preserved.
+
+More formally, for all `0 <= id < jd < dim(inputs[0], d)`, either
+`compare_i_j = compare_j_i = false` or `compare_i_j = true`, where:
+  1. `compare_i_j` $=$ `comparator(inputs[0][i], inputs[0][j], inputs[1][i], inputs[1][j], ...)`.
+  1. For all indices `i = [i0, ..., iR-1]` and `j = [j0, ..., jR-1]`.
+  1. Where `i` $=$ `j` everywhere except for the `d`th dimension.
+  1. Where `d` $=$ `dimension >= 0 ? dimension : rank(inputs[0]) + dimension`.
+
+### Inputs
+
+| Name         | Type                                              |
+|--------------|---------------------------------------------------|
+| `inputs`     | variadic number of tensors of any supported types |
```

Comment:
Should be singular `type` as it could be interpreted as tensors having different supported types for the tensors. We'll need to fix this for other ops.
`variadic number of tensors of any supported type`